### PR TITLE
chore: update flux cli version

### PR DESCRIPTION
**Pull request by toledompm**
## Context
This PR fixes pinned versions and outdated repositories for some of our dependencies, allowing us to build the IAC image again in order to update flux-cli to the latest version

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,8 +42,16 @@ RUN apk update && \
       openssh=${OPENSSH_VERSION} \
       kustomize=${KUSTOMIZE_VERSION}
 
-RUN apk add --no-cache helm aws-cli yq --repository=https://dl-cdn.alpinelinux.org/alpine/edge/community
-RUN apk add --no-cache opa flux --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing
+RUN apk add --no-cache \
+      helm \
+      aws-cli \
+      yq \
+      flux \
```
Comment (toledompm) [inline Dockerfile:49]: Flux was moved to the community repository

File: Dockerfile [hunk 2]
```diff
@@ -42,8 +42,16 @@ RUN apk update && \
...
+      --repository=https://dl-cdn.alpinelinux.org/alpine/edge/community
+
+# OPA
+RUN curl -L -o /usr/bin/opa https://github.com/open-policy-agent/opa/releases/download/v0.11.0/opa_linux_amd64 \
+    && chmod +x /usr/bin/opa
```
Comment (toledompm) [inline Dockerfile:54]: OPA was removed from the alpine packages

File: Dockerfile [hunk 3]
```diff
@@ -12,7 +12,7 @@ ARG PYTHON_VERSION=3.12.3-r1
 ARG PY3_PIP_VERSION=24.0-r2
 ARG ZIP_VERSION=3.0-r12
 ARG OPENSSH_VERSION=9.7_p1-r4
-ARG KUSTOMIZE_VERSION=5.3.0-r5
+ARG KUSTOMIZE_VERSION=5.3.0-r6
```
Comment (toledompm) [inline Dockerfile:15]:
```
1.135 ERROR: unable to select packages:
1.136   curl-8.9.0-r0:
1.136     breaks: world[curl=8.8.0-r0]
1.136   kustomize-5.3.0-r6:
1.136     breaks: world[kustomize=5.3.0-r5]
```

curl and kustomize were not able to install at the current versions when installing locally

Comment (Kasama) [reply Dockerfile:15]: not sure I understand this. It won't work on `8.9.0` and `-r6`?

**Review by Kasama — APPROVED** [review]

**Comment by fdpeiter** [comment]
Certeza que tu quer aprovar @Kasama ?

File: Dockerfile [hunk 4]
```diff
@@ -42,8 +42,16 @@ RUN apk update && \
...
-RUN apk add --no-cache opa flux --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing
+RUN apk add --no-cache \
+      helm \
+      aws-cli \
+      yq \
+      flux \
+      --repository=https://dl-cdn.alpinelinux.org/alpine/edge/community
+
+# OPA
+RUN curl -L -o /usr/bin/opa https://github.com/open-policy-agent/opa/releases/download/v0.11.0/opa_linux_amd64 \
```
Comment (toledompm) [inline Dockerfile:53]:
```suggestion
RUN curl -L -o /usr/bin/opa https://github.com/open-policy-agent/opa/releases/download/v0.67.0/opa_linux_amd64 \
```

**Pull request merged**